src/components/source_editor/LatticeConfigurationDialog.js

```diff
@@ -162,7 +190,7 @@ class LatticeConfigurationDialog extends React.Component {
                                     <label className="fg-label">Lattice 'b'</label>
                                     <input type="number" name="b" className="form-control fc-alt fg-input" min="0"
                                         step="0.05"
-                                        disabled={this.isDisabled('b')}
```

Comment:
isDisabled funciton always returns `false` in current implementation - see line 67-71